UT4MasterServer/UT4MasterServer.csproj

```
@@ -11,12 +11,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.17.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.11" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.20.1" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.16" />
     <PackageReference Include="Serilog.Enrichers.Thread" Version="3.1.0" />
     <PackageReference Include="Serilog.Extensions.Logging" Version="3.1.0" />
     <PackageReference Include="Serilog.Settings.Configuration" Version="3.4.0" />
```

```suggestion
    <PackageReference Include="Serilog.Extensions.Logging" Version="8.0.0" />
    <PackageReference Include="Serilog.Settings.Configuration" Version="8.0.0" />
```

Changelogs:
https://github.com/serilog/serilog-extensions-logging/releases
https://github.com/serilog/serilog-settings-configuration/releases

---

I don't think we should upgrade these because they depend on `Microsoft.Extensions.*` version 8.0.0. Which would cause a mismatch in major versions as we previously discussed.

See here: https://www.nuget.org/packages/Serilog.Extensions.Logging/8.0.0?_src=template#dependencies-body-tab

---

We should probably upgrade whole solution to .NET 8 soon to keep up with the c# ecosystem.

---

OK. I will update our runtime in near future